cobalt/media/base/sbplayer_pipeline.cc

```diff
@@ -398,6 +443,8 @@ SbPlayerPipeline::SbPlayerPipeline(
                     kSbPlayerStateInitialized,
                     "The underlying SbPlayer state of the media pipeline."),
       decode_target_provider_(decode_target_provider),
+      audio_write_duration_local_(audio_write_duration_local),
+      audio_write_duration_remote_(audio_write_duration_remote),
```

Comment:
So, as it is now, `audio_write_duration_local_` and `audio_write_duration_remote_` are ignored when the extension is disabled. Since they are `const`, I'm forced to set their values in the initializer list, unless I make them non-const.

Reply:
Both setting the value in initializer list or remove `const` are ok. Note that we want to set `audio_write_duration_local_` and `audio_write_duration_remote_` to `audio_write_duration_local` and `audio_write_duration_remote` only when the extension is enabled.

Reply:
Sure, I've removed const.